2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,58 @@
+#__BY:sirjan13__
+
+
+def gcd(n, m):
+    if n == 0:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpnnz9__p2/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpnnz9__p2/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -26,7 +26,6 @@
     return ans
 
 
- 
 mod1 = 10**9+7
 mod2 = mod1 + 2
 for cppp in range(int(raw_input())):
```